spec/graphql/types/vote_type_spec.rb

```diff
@@ -3,11 +3,10 @@
 describe Types::VoteType do
   it "only returns date and hour for created_at" do
     created_at = Time.zone.parse("2017-12-31 9:30:15")
-    create(:vote, created_at: created_at)
+    vote = create(:vote, created_at: created_at)
 
-    response = execute("{ votes { edges { node { public_created_at } } } }")
-    received_timestamps = extract_fields(response, "votes", "public_created_at")
+    response = run_graphql_field("Vote.public_created_at", vote)
 
-    expect(Time.zone.parse(received_timestamps.first)).to eq Time.zone.parse("2017-12-31 9:00:00")
+    expect(Time.zone.parse(response.to_s)).to eq Time.zone.parse("2017-12-31 09:00:00")
```

Comment:
It looks like `run_graphql_field` doesn't work like we thought it would :relieved:.

In `Types::VotesType` we define:

```ruby
field :public_created_at, String, null: true
```

However,  `run_graphql_field` is returning an `ActiveSupport::TimeWithZone` object instead of a `String`, which means it isn't really testing our code :thinking:.

I'd suggest leaving it like it was before these changes until `run_graphql_field` takes the defined type into account.